frontend/src/Admin/actions.js

```diff
@@ -137,3 +137,15 @@ export const downloadZip = showId => (dispatch, getState, client) => {
     '_self'
   )
 }
+
+export const downloadCsv = showId => (dispatch, getState, client) => {
+  const { shared: { auth: { downloadToken } } } = getState()
+
+  // TODO replace this with the deployed url
```

Comment:
After #239 gets merged in, you should make this URL a constant.

Reply:
constant made in last commit